### PR TITLE
Add Bundler 2.6 to docs site, and a release blog post

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
### What was the end-user problem that led to this PR?

The problem was that Bundler 2.6 is still not included in the documentation site, and that we should present the new stuff in Bundler 2.6.

### What is your fix for the problem, implemented in this PR?

My fix is to add the new version to documentation and write a blog post about it.

### CODE REVIEW

**Review by simi — APPROVED** [review]

File: source/blog/2024-12-18-bundler-v2-6.html.markdown [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+--normalize-platforms`.
+
+## Other notable changes in Bundler 2.6
+
+This is a non-exhaustive list of other notable improvements in Bundler 2.6:
+
+### Better support for switching between different versions of Ruby
+
+Sometimes when switching between different Ruby versions, even if the resolution
+recorded in the Gemfile.lock is stable, the most optimal platform specific
```
Comment (olleolleolle) [inline source/blog/2024-12-18-bundler-v2-6.html.markdown:111]:
```suggestion
recorded in the Gemfile.lock is stable, the most optimal platform-specific
```

File: source/blog/2024-12-18-bundler-v2-6.html.markdown [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+version for a gem may change. For example, `nokogiri-1.18.0.rc1-x86_64-darwin`
+does not support Ruby 3.5, but `nokogiri-1.18.0.rc1` does. Bundler 2.6 should be
+able to keep a consistent lockfile with all platform specific variants, and use
```
Comment (olleolleolle) [inline source/blog/2024-12-18-bundler-v2-6.html.markdown:114]:
```suggestion
able to keep a consistent lockfile with all platform-specific variants, and use
```

File: source/blog/2024-12-18-bundler-v2-6.html.markdown [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+the one that works best for each Ruby versiom.
```
Comment (olleolleolle) [inline source/blog/2024-12-18-bundler-v2-6.html.markdown:115]:
```suggestion
the one that works best for each Ruby version.
```

File: source/blog/2024-12-18-bundler-v2-6.html.markdown [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+### Better documentation of commands and CLI flags
+
+We have introduced changes to make sure that this site, and the `bundle` CLI,
+properly documents all commands and CLI flags, and made sure it will stay like
+that going forward.
+
+And like always, many other bug fixes and improvements were implemented. Check
+the full changelog
+[here](https://github.com/rubygems/rubygems/releases/tag/bundler-v2.6.0) for
+details.
```
Comment (olleolleolle) [inline source/blog/2024-12-18-bundler-v2-6.html.markdown:144]:
```suggestion
And like always, many other bug fixes and improvements were implemented. Check
[the full Bundler 2.6 changelog](https://github.com/rubygems/rubygems/releases/tag/bundler-v2.6.0) for
details.
```

**Review by olleolleolle — APPROVED** [review]
Thanks for blogging so well!

**Review by tnir — CHANGES_REQUESTED** [review]
@deivid-rodriguez Thanks for updating, but we need to update https://bundler.io/compatibility.html (source: https://github.com/rubygems/bundler-site/blob/master/source/compatibility.html.haml)

**Comment by deivid-rodriguez** [comment]
> Thanks for blogging so well!

Haha, not so sure about that... 😅, but thank you!

> @deivid-rodriguez Thanks for updating, but we need to update https://bundler.io/compatibility.html (source: https://github.com/rubygems/bundler-site/blob/master/source/compatibility.html.haml)

Nice catch, let me update that now!

**Comment by deivid-rodriguez** [comment]
I'll update blog post date and publish this tomorrow. If @tnir doesn't get to it, we can iterate after merge if everything else is needed.

**Comment by tnir** [comment]
@deivid-rodriguez Thanks 🌮 and sorry for the delayed response! 🎉

**Comment by deivid-rodriguez** [comment]
No problem, thanks for your review!

**Pull request merged**